addons/bestja_stores/models.py

```diff
@@ -162,27 +203,247 @@ class StoreInProject(models.Model):
         ('deactivated', "dezaktywowany"),
     ]
 
-    store = fields.Many2one('bestja_stores.store', required=True)
-    project = fields.Many2one('bestja.project', required=True)
+    def _default_project(self):
+        if self.env.context.get('active_model') == 'bestja.project':
+            return self.env.context.get('active_id')
+        return self.env.context.get('default_project')
+
+    store = fields.Many2one(
+        'bestja_stores.store',
+        required=True,
+        domain='''[
+            ('state', '=', 'accepted'),
+            ('default_partner', '=', organization),
+            ('id', '__free_for_project__', project),
+        ]''',  # a custom operator defined in Store's _search method
```

Comment:
Shouldn’t this comment be put in the line `('id', '__free_for_project__', project),` or above?